parm/config/gfs/config.base.emc.dyn

```diff
@@ -181,6 +181,10 @@ case "${CASE}" in
         export OCNRES=025
         export waveGRD='mx025'
         ;;
+    "C1152")
+        export OCNRES=025
+        export waveGRD='mx025'
+        ;;
```

Comment:
Since this is identical to C768, instead of repeating, please update line 180 as:
```
"C768" | "C1152")

Reply:
@aerorahul fixed in config.base.emc.dyn